locations/spiders/bakers_baristas_gb_ie.py

```
@@ -1,12 +1,22 @@
     name = "bakers_baristas_gb_ie"
     item_attributes = {
         "brand_wikidata": "Q85199581",
         "brand": "Bakers + Baristas",
     }
-    allowed_domains = [
-        "www.bakersbaristas.com",
-    ]
+    # allowed_domains = [
+    #     "www.bakersbaristas.com",
+    # ]
+    start_urls = ["https://matthews42.sg-host.com/test-location-app/store-locations.php"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)